selfdrive/car/tests/test_models.py

```diff
@@ -214,10 +214,10 @@ def test_panda_safety_carstate(self):
 
     # warm up pass, as initial states may be different
     for can in self.can_msgs[:300]:
+      self.CI.update(CC, (can.as_builder().to_bytes(), ))
```

Comment:
@adeebshihadeh do you remember why we passed in a single CAN message at a time here?